frontend/talentsearch/src/js/components/search/SearchPools.tsx

```diff
@@ -32,8 +32,11 @@ const SearchPools: React.FunctionComponent<SearchPoolsProps> = ({
     pool.classifications as SimpleClassification[];
 
   return (
-    <div data-h2-padding="base(x1)">
-      <p data-h2-font-weight="base(700)">
+    <article
+      data-h2-padding="base(x1)"
+      aria-labelledby={`search_pool_${pool.id}`}
+    >
+      <p data-h2-font-weight="base(700)" id={`search_pool_${pool.id}`}>
```

Comment:
why not use `data-test` attr, which is a cypress recommended way (that we mostly ignore).
https://docs.cypress.io/guides/references/best-practices#How-It-Works

The nice part about this is that there are already webpack plugins to filter any of this out of prod builds (if we want), whereas using this is hard to strip later.

```suggestion
      data-test={`search_pool_${pool.id}`}
    >
      <p data-h2-font-weight="base(700)" data-test={`search_pool_${pool.id}`}>
```

Reply:
hm.... I'm not seeing "search_pool_" anywhere else. Did this end up being used in selectors?